reliability/level4/README.md

```diff
@@ -0,0 +1,38 @@
+# Level 4
+
+## Summary
+
+The level4 program is the same as level2 and level3.
+
+## Instructions
+
+Write a simple HTTP server that serves the requests, parse and "enrich" the logs, and store them into a Redis `LIST` on a local Redis instance.
+
+The log "enrichment" is done using the provided `SlowComputation` module.
+
+**BEWARE** This module is pretty slow and the level4 program enforces a 100ms timeout.
```

Comment:
may be we can add something like "this means that any request lasting more than this timeout will fail, so we want an implementation that will avoid it."

Reply:
because it is not that clear, if you don't know the issue, in my opinion